test/cmdlineTests/optimizer_BlockDeDuplicator/output

```diff
@@ -66,24 +66,24 @@ sub_0: assembly {
       dup1
       0x2e1fb2bc
       eq
-      tag_3
+      tag_4
       jumpi
       dup1
       0x4753a67d
       eq
-      tag_3
+      tag_4
       jumpi
     tag_2:
       0x00
       dup1
       revert
         /* "optimizer_BlockDeDuplicator/input.sol":138:174  function f() public { true ? 1 : 3;} */
     tag_3:
-      tag_6
-      tag_7
-      jump	// in
-    tag_6:
       stop
+        /* "optimizer_BlockDeDuplicator/input.sol":108:133  function fun_() public {} */
+    tag_4:
+      jump(tag_3)
```

Comment:
This could actually be replaced by a single `stop`...

Reply:
Yes - in general we can consider also inlining ``pushtag(tag) jump`` when we see ``tag: ... stop`` or ``tag: ... invalid`` or ``tag: ... revert``... it'd need a different heuristics, though.

Reply:
Of course the size of the block to be inlined being smaller than the ``pushtag(tag) jump`` itself will always be good...

Reply:
I think it would be useful, even if again not that nice for the debugger. But maybe we can do that in another PR.